rmf_workcell_editor/Cargo.toml

```
@@ -0,0 +1,36 @@
+[package]
+name = "rmf_workcell_editor"
+version = "0.0.1"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+name = "librmf_workcell_editor"
+
+[[bin]]
+path = "src/main.rs"
+name = "rmf_workcell_editor"
+
+[dependencies]
+# PR merged after 0.10 but not released yet, bump to 0.10.1 once merged
+bevy_infinite_grid = { git = "https://github.com/ForesightMiningSoftwareCorporation/bevy_infinite_grid", rev = "86018dd" }
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+wasm-bindgen = "=0.2.93"
+bevy = { version = "0.12", features = ["pnm", "jpeg", "tga"] }
+thiserror = "*"
+rmf_workcell_format = { path = "../rmf_workcell_format", features = ["bevy_support"] }
+# rmf_site_editor = { git = "https://github.com/open-rmf/rmf_site", rev = "fe86373"}
+rmf_site_editor = { git = "https://github.com/open-rmf/rmf_site", branch = "luca/private_run_js", default-features = false}
```

I've created a `v0.0.1` tag for the site editor which has everything that was merged before https://github.com/open-rmf/rmf_site/pull/242 so I believe it should be suitable for the workcell editor, so we should be able to change this to

```suggestion
rmf_site_editor = { git = "https://github.com/open-rmf/rmf_site", tag = "v0.0.1", default-features = false}
```

If we do need to patch anything from v0.0.1 we can branch off of the commit for that tag and then make a new tag called `v0.0.1-2` or something like that. Meanwhile we'll bump the `main` branch to v0.0.2 once https://github.com/open-rmf/rmf_site/pull/242 is merged.